### PR TITLE
F5 BIG AWAF Pack

### PR DESCRIPTION
**Pull request by eepstain**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/CIAC-9402)

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/F5BigIPAWAF/README.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+* The product documentation is available [here](https://techdocs.f5.com/kb/en-us/products/big-ip_asm/manuals/product/asm-implementations-11-6-0/14.html). 
+
+### Creating a logging profile
+1. On the Main tab, click **Security** &rarr; **Event Logs** &rarr; **Logging Profiles**.
+2. Click **Create**.
+3. In the **Profile Name** field, type a unique name for the profile.
+4. Select the **Application Security** check box.
+5. On the **Application Security** tab, for **Configuration**, select **Advanced**.
+6. Select the **Remote Storage** check box.
```
Comment (ShirleyDenkberg) [inline Packs/F5BigIPAWAF/README.md:19]:
```suggestion
6. Select the **Remote Storage** checkbox.
```

File: Packs/F5BigIPAWAF/README.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+7. Click on **Finished**.
```
Comment (ShirleyDenkberg) [inline Packs/F5BigIPAWAF/README.md:20]:
```suggestion
7. Click **Finished**.
```

File: Packs/F5BigIPAWAF/README.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+In order to do so, create a logging profile and set remote logging to the relevant server.
+
+* The product documentation is available [here](https://techdocs.f5.com/kb/en-us/products/big-ip_asm/manuals/product/asm-implementations-11-6-0/14.html). 
+
+### Creating a logging profile
+1. On the Main tab, click **Security** &rarr; **Event Logs** &rarr; **Logging Profiles**.
+2. Click **Create**.
+3. In the **Profile Name** field, type a unique name for the profile.
+4. Select the **Application Security** check box.
+5. On the **Application Security** tab, for **Configuration**, select **Advanced**.
```
Comment (ShirleyDenkberg) [inline Packs/F5BigIPAWAF/README.md:18]:
```suggestion
5. On the *Application Security* tab, for *Configuration*, select **Advanced**.
```

File: Packs/F5BigIPAWAF/README.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+6. Select the **Remote Storage** check box.
+7. Click on **Finished**.
+
+### Setting Remote Logging
+1. Connect to the BIG-IP web UI and login with administrative rights.
```
Comment (ShirleyDenkberg) [inline Packs/F5BigIPAWAF/README.md:23]:
```suggestion
1. Connect to the BIG-IP web UI and log in with administrative rights.
```

File: Packs/F5BigIPAWAF/README.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+2. Navigate to **Security** &rarr; **Event Logs** &rarr; **Logging Profiles**.
+3. Click the name of the logging profile for which you want to set up remote logging.
+4. Select the **Remote Storage** check box.
```
Comment (ShirleyDenkberg) [inline Packs/F5BigIPAWAF/README.md:26]:
```suggestion
4. Select the **Remote Storage** checkbox.
```

File: Packs/F5BigIPAWAF/README.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+5. From the **Remote Storage Type** list, select **Remote**.
```
Comment (ShirleyDenkberg) [inline Packs/F5BigIPAWAF/README.md:27]:
```suggestion
5. From the *Remote Storage Type* list, select **Remote**.
```

File: Packs/F5BigIPAWAF/README.md [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+6. For the **Protocol** setting, select **TCP**.
```
Comment (ShirleyDenkberg) [inline Packs/F5BigIPAWAF/README.md:28]:
```suggestion
6. For the *Protocol* setting, select **TCP**.
```

File: Packs/F5BigIPAWAF/README.md [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+7. For **Server Addresses**, Type the IP Address, Port Number (default is 514), and click **Add**.
```
Comment (ShirleyDenkberg) [inline Packs/F5BigIPAWAF/README.md:29]:
```suggestion
7. For *Server Addresses*, type the IP Address, Port Number (default is 514), and click **Add**.
```

File: Packs/F5BigIPAWAF/README.md [hunk 9]
```diff
@@ -0,0 +1,77 @@
...
+8. Click on **Finished**.
```
Comment (ShirleyDenkberg) [inline Packs/F5BigIPAWAF/README.md:30]:
```suggestion
8. Click **Finished**. 
```

File: Packs/F5BigIPAWAF/README.md [hunk 10]
```diff
@@ -0,0 +1,77 @@
...
+3. Click the name of the logging profile for which you want to set up remote logging.
+4. Select the **Remote Storage** check box.
+5. From the **Remote Storage Type** list, select **Remote**.
+6. For the **Protocol** setting, select **TCP**.
+7. For **Server Addresses**, Type the IP Address, Port Number (default is 514), and click **Add**.
+8. Click on **Finished**. 
+
+### Supported Timestamp Ingestion 
+Timestamp ingestion is supported for the format: **%Y-%m-%dT%H:%M:%S%Ez** (yyyy-mm-ddTHH:MM:SS+ZZ:ZZ).
+In order to configure the required timestamp for Syslog messages follow these instructions:
```
Comment (ShirleyDenkberg) [inline Packs/F5BigIPAWAF/README.md:34]:
```suggestion
In order to configure the required timestamp for syslog messages, follow these instructions:
```

File: Packs/F5BigIPAWAF/README.md [hunk 11]
```diff
@@ -0,0 +1,77 @@
...
+
+* The product documentation is available [here](https://my.f5.com/manage/s/article/K02138840). 
+
+1. Login to BIG-IP command line.
```
Comment (ShirleyDenkberg) [inline Packs/F5BigIPAWAF/README.md:38]:
```suggestion
1. Log in to the BIG-IP command line.
```

File: Packs/F5BigIPAWAF/README.md [hunk 12]
```diff
@@ -0,0 +1,77 @@
...
+2. Use Linux editor to edit the syslog-ng configuration, in this case using **nano** editor.
```
Comment (ShirleyDenkberg) [inline Packs/F5BigIPAWAF/README.md:39]:
```suggestion
2. Use a Linux editor to edit the syslog-ng configuration, in this case using *nano* editor.
```

File: Packs/F5BigIPAWAF/README.md [hunk 13]
```diff
@@ -0,0 +1,77 @@
...
+## Collect Events from Vendor
+In order to use the collector, use the [Broker VM](#broker-vm) option.
+ 
+### Broker VM
+To create or configure the Broker VM, use the information described [here](https://docs-cortex.paloaltonetworks.com/r/Cortex-XDR/Cortex-XDR-Pro-Administrator-Guide/Configure-the-Broker-VM).
+ 
+You can configure the specific vendor and product for this instance.
+ 
+1. Navigate to **Settings** &rarr; **Configuration** &rarr; **Data Broker** &rarr; **Broker VMs**.
+2. Go to the **Apps** column under the **Brokers** tab and add the **Syslog Collector** app for the relevant broker instance. If the app already exists, hover over it and click **Configure**.
```
Comment (ShirleyDenkberg) [inline Packs/F5BigIPAWAF/README.md:69]:
```suggestion
2. Go to the *Apps* column under the *Brokers* tab and add the *Syslog Collector* app for the relevant broker instance. If the app already exists, hover over it and click **Configure**.
```

**Comment by ShirleyDenkberg** [comment]
@eepstain Doc review completed.

**Review by OBavly — APPROVED** [review]

**Pull request merged**